frontend/src/Logger/Logger.tsx

```diff
@@ -1,11 +1,11 @@
-class Logger {
-    static log(message: string, ...optionalParams: any[]) {
+export class Logger {
+    static log(message: string, ...optionalParams: unknown[]) {
```

Comment:
just a quick question: 
Does it have to be "unknown"?
I don´t know, that's why I´m asking :)